.github/workflows/add-changelog-item.yml

```diff
@@ -0,0 +1,31 @@
+name: Add changelog item
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  add-changelog-item:
+    runs-on: ubuntu-latest
+    container: node:14
```

Comment:
Can you take a look at how the ci workflow is setup and mimic that. That way we can use the latest version of node.

Reply:
Looks good.